### PR TITLE
Migrate soak jobs to bootstrap

### PR DESCRIPTION
**Pull request by fejta**
ref #1021 #1028 

FYI Kris and Saad this includes migrating the 1.5 soak jobs.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1107)

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    - workspace-cleanup:
+        dirmatch: true
+        exclude:
+        - 'test-infra/.git/'
+        external-deletion-command: 'sudo rm -rf %s'
+
+- project:
+    jobs:
+    - 'ci-{soak-suffix}'
+    name: bootstrap-ci
```
Comment (krzyzacy) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml]: `jenkins_jobs.errors.JenkinsJobsException: Duplicate entry found in '/test-infra/jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml: 'bootstrap-ci' already defined`

this need to be bootstrap-ci-soak?

Comment (fejta) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml]: Done

**Comment by krzyzacy** [comment]
why the bot did not give me the actual label this time... @spxtr

**Comment by fejta** [comment]
I think the command needs to be on its own line to distinguish from lgtm cancel etc

**Comment by krzyzacy** [comment]
fair enough..

**Pull request merged**